proto/dwtools/abase/l0.test/Long.test.s

```diff
@@ -13427,14 +13427,80 @@ function longFromRange( test )
   });
 
 };
+
 //
 
-function longFromRange1( test )
+function longFromRangeWithStep( test )
 {
+  test.case = 'equal numders of range';
+  var got = _.longFromRangeWithStep( [ 1, 1 ], 2 );
+  var expected = [ ];
+  test.identical( got, expected );
 
-  console.log('Hello world');
+  test.case = 'valid range with float step';
+  var got = _.longFromRangeWithStep( [ 0, 10 ], 1.5 );
+  var expected = [ 0, 1.5, 3, 4.5, 6, 7.5, 9 ];
+  test.identical( got, expected );
+
+  test.case = 'decrementing range with step < 0';
+  var got = _.longFromRangeWithStep( [ 10, 0 ], -1 );
+  var expected = [ 10, 9, 8, 7, 6, 5, 4, 3, 2, 1 ];
+  test.identical( got, expected );
+
+  test.case = 'step is undefined';
+  var got = _.longFromRangeWithStep( [ 4, 6 ], undefined );
+  var expected = [ 4, 5 ];
+  test.identical( got, expected );
+
+  test.case = 'step is undefined and first number bigger';
+  var got = _.longFromRangeWithStep( [ 10, 0 ], undefined );
+  var expected = [ 10, 9, 8, 7, 6, 5, 4, 3, 2, 1 ];
+  test.identical( got, expected );
+
+  test.case = 'decrementing range with step < 0';
```

Comment:
Please, add a test case with a negative float step. Description should have one style.